dts/bindings/display/display-controller.yaml

```diff
@@ -18,3 +18,11 @@ properties:
 
     label:
       required: true
+
+    backlight-led:
```

Comment:
Generally, a display controller is not a display (even if for smaller displays like ST7789V one can say it is a unit). A display (panel) can have a backlight, what does not necessarily have to be a LED, or none at all (OLED, EPD). So this does not fit here. Probably you can put it in ST7789V bindings.

Reply:
You could use the same arguments for height and width, but lets not discuss that in this issue. 

I moved the property to the ST7789V binding.